kfs-core/src/main/java/edu/arizona/kfs/coa/businessobject/AccountExtension.java

```diff
@@ -89,6 +90,10 @@ public void setFaCostSubCatCode(String faCostSubCatCode) {
         this.faCostSubCatCode = faCostSubCatCode;
     }
 
+    public FACostSubCategory getFaCostSubCategory() {
+    	return faCostSubCategory;
+    }
+

```

Comment:
Where is setter method for faCostSubCategory object? Setter and getter methods for faCostSubCategory object need to be more robust. See setter and getter methods for budgetShell object.